src/lib/isAlpha.js

```diff
@@ -17,7 +17,14 @@ export default function isAlpha(_str, locale = 'en-US', options = {}) {
     }
   }
 
-  if (locale in alpha) {
+  if (Array.isArray(locale)) {
+    const regex = locale.map(localeKey => alpha[localeKey])
+      .filter(localeRegex => !!localeRegex)
+      .map(localeRegex => localeRegex.source.replace(/\^|\$/g, ''))
```

Comment:
```suggestion
    const regex = locale.map(localeKey => {
      if (localeKey in alpha) {
        return localeRegex.source.replace(/\^|\$/g, '')
      }
    })
```

Reply:
map should always return, so this wont work. Filter should handle any invalid locale in the original code.